assets/js/src/core/modules/element/actions/lock/use-lock.tsx

```diff
@@ -36,6 +39,8 @@ export interface UseLockHookReturn {
 }
 
 export const useLock = (elementType: ElementType): UseLockHookReturn => {
+  const state = store.getState()
+  const user = selectCurrentUser(state as { [userSliceName]: UserInformation })
```

Comment:
Please use the `useUser` use hook which you created some time ago to retrieve the user. This would simplify and centralize the approach a bit.